<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>617</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>565</width>
    <height>558</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>RTS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">selection-color: rgb(108, 108, 108);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>571</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tabTernario">
     <attribute name="title">
      <string>Ternario</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/imagenes/rts.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>166</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarPalabraT">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>80</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listaT">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>221</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listaPrefijoT">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>201</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar para borrar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaAgregarT">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaBuscarT">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaPrefijoT">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>147</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarT">
      <property name="geometry">
       <rect>
        <x>424</x>
        <y>50</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="borrarT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>110</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="barraDeEstadoT">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>456</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deshacerT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>419</x>
        <y>110</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Deshacer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>126</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar prefijo:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAbrirT">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imagenes/documents07.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonGuardarT">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imagenes/floppy.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonGuardarComoT">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imagenes/disque10.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRway">
     <attribute name="title">
      <string>R-Way</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>166</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buscarPalabraR">
      <property name="geometry">
       <rect>
        <x>425</x>
        <y>80</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listaR">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>221</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listaPrefijoR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>201</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>258</x>
        <y>110</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Seleccionar para borrar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaAgregarR">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaBuscarR">
      <property name="geometry">
       <rect>
        <x>262</x>
        <y>80</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="entradaPrefijoR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>147</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="agregarR">
      <property name="geometry">
       <rect>
        <x>424</x>
        <y>50</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="borrarR">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>372</x>
        <y>110</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="text">
       <string>Borrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="barraDeEstadoR">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>456</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="deshacerR">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>419</x>
        <y>110</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Deshacer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>126</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar prefijo:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="botonGuardarR">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imagenes/floppy.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonAbrirR">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/imagenes/documents07.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="botonGuardarComoR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/imagenes/disque10.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/imagenes/rts.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSalir">
    <property name="title">
     <string>Salir</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <addaction name="menuSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSalir"/>
  </widget>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="imagenes.qrc">
     <normaloff>:/imagenes/salir-de-gnome-icono-7637-96.png</normaloff>:/imagenes/salir-de-gnome-icono-7637-96.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
